package.json

```diff
@@ -83,83 +83,83 @@
     ]
   },
   "dependencies": {
-    "@ministryofjustice/frontend": "^1.0.0",
-    "agentkeepalive": "^4.1.4",
-    "applicationinsights": "^2.1.8",
-    "body-parser": "^1.19.0",
+    "@ministryofjustice/frontend": "^1.2.0",
+    "agentkeepalive": "^4.2.0",
+    "applicationinsights": "^2.2.1",
+    "body-parser": "^1.19.1",
     "bunyan": "^1.8.15",
     "bunyan-format": "^0.2.1",
     "compression": "^1.7.4",
     "connect-flash": "^0.1.1",
     "connect-redis": "^6.0.0",
-    "cookie-session": "^1.4.0",
+    "cookie-session": "^2.0.0",
     "csurf": "^1.11.0",
-    "dotenv": "^10.0.0",
-    "express": "^4.17.1",
+    "dotenv": "^16.0.0",
+    "express": "^4.17.2",
     "express-request-id": "^1.4.1",
```

Comment:
This was originally bumped to `"express-request-id": "^2.0.1"` however when running locally I got the error
` Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: hmpps-registers/node_modules/express-request-id/index.js`

Which was curious so I started looking into the version history of `express-request-id` - they've started dev on `2.0.0` this year but looks like [DefinitelyTyped/express-request-id](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/express-request-id) hasn't been updated

Seems ok to leave as it is given it doesn't raise npm audit issue